phase-4/README.md

```diff
@@ -0,0 +1,18 @@
+# Phase 4
+:::tip Phase 4 starting soon
+Phase 4 is going to start soon  
+No more submissions are accepted.    
+:::
+
```

Comment:
I will export `morpheus-3000` at height `820,300` which I estimate to be around `24 April 2020 0300 UTC`. If we want to give more time for validators to prepare, we can set we only start calculating the precommits from `27 April 2020 0500 UTC` which will be on Monday. And it already covers different working patterns over all timezones for 3 full days.